project/UI/CarbonPage/CarbonPage.swift

```diff
@@ -22,26 +22,26 @@ public struct CarbonPageCommand: ParsableCommand {
     @ArgumentParser.Option(help: ArgumentHelp("Path where Carbon snippets are saved to. ex. `/home`", valueName: "carbon output"))
     private var output: ArgumentPath
     
-    @ArgumentParser.Option(default: "nef", help: "Background color in hexadecimal")
-    private var background: String
+    @ArgumentParser.Option(help: "Background color in hexadecimal")
```

Comment:
This help message is a bit misleading, as you can also use named colors.

Reply:
With auto-completion it will suggest you use one of the predefined colors, but do you have any suggestion to message?  - I was thinking about to a simple `Background color` but I think it is better to add the expected format, so maybe we could say `Background color in hexadecimal or predefined` wdut?